src/main/java/application/bookstore/repository/book/BookSpecificationBuilder.java

```diff
@@ -0,0 +1,34 @@
+package application.bookstore.repository.book;
+
+import application.bookstore.dto.BookSearchParameters;
+import application.bookstore.model.Book;
+import application.bookstore.repository.SpecificationBuilder;
+import application.bookstore.repository.SpecificationProviderManager;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.jpa.domain.Specification;
+import org.springframework.stereotype.Component;
+
+@RequiredArgsConstructor
+@Component
+public class BookSpecificationBuilder implements SpecificationBuilder<Book> {
+    private static final String TITLE = "title";
+    private static final String AUTHOR = "author";
```

Comment:
You already have these constants in SpecificationProvider classes, don`t duplicate code